en/docs/getting-started/key-concepts.md

```diff
@@ -1,25 +1,22 @@
-| <a name="tenant"></a> Tenant                             | A tenant in WSO2 API Manager is a logically isolated entity. A tenant can be mapped as an organizational unit, department, etc. Multi-tenancy enables such organizational units/departments to share the same API Manager deployment and the respective resources. But function individually with a personalized view/scope of the installation. Multi-tenancy is popularly used in SaaS offerings. |
-|  <a name="api-publisher-role"></a> API Publisher Role                 | An API publisher (API provider) is an API product manager. He/she manages a set of APIs and API products across the enterprise or business unit and controls the API lifecycle, subscriptions, and monetization aspects of an API or API product. The API Publisher can also analyze usage patterns for APIs and access all APIs statistics. |
-|  <a name="api-creator-role"></a> API Creator Role                   | An API Creator is a person in a technical role who understands the technical aspects of the API (interfaces, documentation, versions, etc.) and uses the API Publisher portal for the design and development of APIs.  APIs created by an API creator are managed (lifecycle managed and productized) by an API product manager. |
-|  <a name="api-subscriber-role"></a> API Subscriber Role                | A subscriber is a consumer of an API. He/she intends to develop one or more applications that consume APIs on the developer portal. A subscriber uses the API developer portal to discover APIs, read the documentation and forums, rate/comment on the APIs, subscribe, obtain keys, and finally use APIs from their applications. |
-|  <a name="admin-role"></a> Admin Role                         | The admin user is responsible for managing the admin portal, which includes tasks such as  approving workflows, managing rate limiting policies, configuring emails for bot detection/alerts, viewing the state and health of APIs, etc. In addition, the admin user is also capable of carrying out general administrative tasks such as managing users, roles, databases, security, etc. |
-
-
-
+|  <a name="api-product"></a> API Product                        | A combination of resources from one or more available APIs. An API product allows API product managers to mix and match resources from the available APIs. Similar to using an API, an application that intends to use an API product needs to subscribe to the APIs under a selected business plan. An application that subscribes to an API product gets access to all of its resources, which belongs to one or more of the APIs that are used to create the API product. |
+|  <a name="access-token"></a> Access Token                       | Used in OAuth2.0 bearer token-based authentication to allow a consumer to access an API. It is a string that is passed as an HTTP header of an API request. WSO2 API Manager supports JWT formatted self-contained access tokens and regular opaque access tokens. |
```

Comment:
@ChamodDamitha , based on https://github.com/wso2/docs-apim/pull/1962 should we remove the following from this sentence?
"and regular opaque access tokens."

Reply:
yes, we should remove